packages/lib/gpcircuits/src/proto-pod-gpc.ts

```diff
@@ -184,7 +245,15 @@ export function protoPODGPCCircuitParamArray(
 export function arrayToProtoPODGPCCircuitParam(
   params: number[]
 ): ProtoPODGPCCircuitParams {
-  return ProtoPODGPCCircuitParams(params[0], params[1], params[2]);
+  return ProtoPODGPCCircuitParams(
+    params[0],
```

Comment:
Optional, but I think `...params` might work here.  A similar thing would work in Python.

Reply:
TypeScript complains if I do this unless I change `ProtoPODGPCCircuitParams` function signature to take a rest parameter, which will probably make things less pretty.